src/manage_configs.c

```diff
@@ -329,6 +339,24 @@ should_sync_config_from_path (const char *path, gboolean rebuild,
   uuid = g_strdup_printf ("%s-%s-%s-%s-%s",
                           split[1], split[2], split[3], split[4], split[5]);
   g_strfreev (split);
+
+  if (resource_id_deprecated ("config", uuid))
+    {
+      find_config_no_acl (uuid, config);
```

Comment:
```suggestion
```

Reply:
Is this line needed ?

Reply:
Yes, the config row id returned via the second parameter is used to check whether it has to be created or updated.